compiler-rt/test/builtins/Unit/extendxftf2_test.c

```diff
@@ -1,10 +1,10 @@
-// RUN: %clang_builtins %s %librt -o %t && %run %t
+// RUN: %clang_builtins %s %librt -fforce-enable-int128 -o %t && %run %t
 // REQUIRES: librt_has_extendxftf2
 
 #include "int_lib.h"
 #include <stdio.h>
 
-#if __LDBL_MANT_DIG__ == 64 && defined(__x86_64__) &&                          \
+#if __LDBL_MANT_DIG__ == 64 && (defined(__x86_64__) || defined(__i386__)) &&   \
     (defined(__FLOAT128__) || defined(__SIZEOF_FLOAT128__))
```

Comment:
```suggestion
#if defined(CRT_HAS_TF_MODE) && HAS_80_BIT_LONG_DOUBLE
```

Since we include int_lib.h should be possible to simplify this